### PR TITLE
Fairfax Australia.js

### PR DESCRIPTION
**Pull request by marcralew**
This translator scrapes all Australian Fairfax media newspapers. The Age, The Sydney Morning Herald, Brisbane times and WAtoday. If accepted it could replace the pre-existing but non-functioning The Age & Sydney Morning Herald translators.

### CODE REVIEW

File: Fairfax Australia [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+itemType	: 'newspaperArticle',
+detect		: FW.Xpath('.//h1[@itemprop="name headline"]'),
+title		: FW.Xpath('.//h1[@itemprop="name headline"]').text().trim(),
+attachments : [ {url: FW.Xpath('//div[@class="article__toolbar"]/ul/li[3]/a').key("href"),
+  title: FW.Xpath('//meta[@name="application-name"]/@content').text().append( " Printable"),
+  type:  "text/html",
+}],
+
+date		: FW.Xpath('//*[@id="content"]/main/article/header/div[1]/ul/li[1]/time').text().trim(),
+creators	: FW.Xpath('//h5[@itemprop="name"]').text().split(/, |" and "/).cleanAuthor("Author"),
```
Comment (adam3smith) [inline Fairfax Australia]: the split doesn't look right to me. I think that should be `.split(/,\s+|\s+and\s+/)`: you can't mix regular expressions with literal expressions.

File: Fairfax Australia [hunk 2]
```diff
@@ -0,0 +1,708 @@
...
+FW.Scraper({
+itemType	: 'newspaperArticle',
+detect		: FW.Xpath('.//h1[@itemprop="name headline"]'),
+title		: FW.Xpath('.//h1[@itemprop="name headline"]').text().trim(),
+attachments : [ {url: FW.Xpath('//div[@class="article__toolbar"]/ul/li[3]/a').key("href"),
+  title: FW.Xpath('//meta[@name="application-name"]/@content').text().append( " Printable"),
+  type:  "text/html",
+}],
+
+date		: FW.Xpath('//*[@id="content"]/main/article/header/div[1]/ul/li[1]/time').text().trim(),
```
Comment (adam3smith) [inline Fairfax Australia]: this looks very fragile to me -- I'd expect the time element is unique within the article header? so couldn't we just use:
`//*[@id="content"]/main/article/header//time`?

Comment (marcralew) [inline Fairfax Australia]: Yes, you are quite right. Code modified.

File: Fairfax Australia (hunk 1 above)
Comment (marcralew) [inline Fairfax Australia]: You are quite correct about mixing regular & literal expressions. Apologies. Code modified.

**Comment by adam3smith** [comment]
Great, thanks. Only change I made was to set the min version to 3.0

**Pull request merged**